Compiler warning

```
src/pray.c: In function ‘goat_eat’:
src/pray.c:4114:1: warning: type of ‘yourinvent’ defaults to ‘int’ [-Wimplicit-int]
 goat_eat(otmp, yourinvent)
```

--- a/src/pray.c
+++ b/src/pray.c
@@ -4119,6 +4119,7 @@ int x, y;
 void
 goat_eat(otmp, yourinvent)
 struct obj *otmp;
+boolean yourinvent;
 {
     int value = 0;
 	struct permonst *ptr = &mons[otmp->corpsenm];

